kano/utils/user.py

```diff
@@ -17,7 +17,7 @@ def get_user_getpass():
 
 
 def get_user():
-    return os.environ['LOGNAME']
+    return os.environ.get('LOGNAME', '')
```

Comment:
Actually, should this return `root` in this case?

Reply:
Not quite sure, seems arbitrary to me as I don't see why not use `getpass.getuser()` for which there is another function... The only place where this seems to be called from is https://github.com/KanoComputing/kano-profile/blob/master/kano_avatar/paths.py#L13

Reply:
I think assuming `root` would be a security harm. I would leave as per Radu's assumption.
If this function fails, surely other things would misbehave in the first place..